mac/RT1.MAC

```diff
@@ -25,12 +25,14 @@
 .* 09/22/06 ADD CCLE AND CCGE OPTIONS FOR USE IN TESTTIM2 TESTS        
 .* 03/05/08 ADD RT1 OK - INC COUNT BY 1                                
 .* 04/19/08 RPI 833 add SETC quotes for HLASM compatibility
+.* 08/17/24 #540 Add parameter to select either WTO or file output
 .*********************************************************************
-&NAME    RT1   &CMD,&CLC1,&CLC2,&LL  
+&NAME    RT1   &CMD,&CLC1,&CLC2,&LL,&TYPE=WTO,&ABEND=101                 #540
 .*
 .* REGRESSION TEST V1 COMMAND MACROS
 .* 
 .* 1.  RT1 MAIN  - START PGM AND DISPLAY STARTUP WTO
+.*                 &TYPE=FILE selects output to file
```

Comment:
Line needs #540 at end. Also, to be consistent with the existing comments, this one should be in all caps.

Reply:
I hate all caps. Leaving this as is.